libvcx/src/credential_def.rs

```diff
@@ -169,7 +208,21 @@ impl CredentialDef {
 
 fn _parse_revocation_details(revocation_details: &str) -> VcxResult<RevocationDetails> {
     serde_json::from_str::<RevocationDetails>(&revocation_details)
-        .to_vcx(VcxErrorKind::InvalidRevocationDetails, "Cannot deserialize RevocationDeltas")
+        .to_vcx(VcxErrorKind::InvalidRevocationDetails, "Cannot deserialize RevocationDetails")
+}
+
+fn _maybe_set_url(rev_reg_def_json: &str, revocation_details: &RevocationDetails, rev_reg_id: &str) -> VcxResult<String> {
+    // TODO: Deserialization into RevRegDef loses ver field
+    let mut rev_reg_def: serde_json::Value = serde_json::from_str(&rev_reg_def_json)
+        .map_err(|err| VcxError::from_msg(VcxErrorKind::InvalidJson, format!("Invalid RevocationRegistryDefinition: {:?}, err: {:?}", rev_reg_def_json, err)))?;
+
+    if let Some(tails_url) = &revocation_details.tails_url {
+        let full_tails_location = vec![tails_url.to_string(), rev_reg_id.to_string()].join("/"); // TODO: Remove trailing slashes
```

Comment:
1. What's the problem with slashes? Is this okay not deal with now? 
2. Why is full tails location vector?

Reply:
1. What do you mean by problem with slashes?
2. It isn't a vector

Reply:
I mean the TODO comment